configure.ac

```diff
@@ -2512,6 +2512,31 @@ fi
 ]
 )
 
+AC_ARG_ENABLE([aria],
+    [AS_HELP_STRING([--enable-aria],[Enable wolfSSL support for ARIA (default: disabled)])],
+    [ ENABLED_ARIA=$enableval ],
+    [ ENABLED_ARIA=no ]
+    )
+if test "$ENABLED_ARIA" = "yes"
+then
+    ARIA_DIR=MagicCrypto
+    # Enable dependency
+    ENABLED_OPENSSLEXTRA="yes"
+    CFLAGS="$CFLAGS -I$ARIA_DIR/include"
+    AM_CFLAGS="$AM_CFLAGS -DHAVE_ARIA -DOPENSSL_EXTRA"
+    AM_LDFLAGS="$AM_LDFLAGS -L$ARIA_DIR/lib -lMagicCrypto"
```

Comment:
Seems like an issue with the library path here? This is Ubuntu 20 and using the MagicCrypto dir.

```
$ ./configure --enable-all --enable-aria
$ make
$ ./scripts/unit.test 
/home/davidgarske/GitHub/wolfssl/tests/.libs/unit.test: error while loading shared libraries: libMagicCrypto.so: cannot open shared object file: No such file or directory
```

Reply:
Yes, you still need to have the `libMagicCrypto.so` in your `LD_LIBRARY_PATH`, similar to what is necessary with `libwolfssl.so`.